.github/workflows/ci.yml

```diff
@@ -0,0 +1,31 @@
+name: Python CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 4
```

Comment:
Any particular reason for imposing this limit?  (Not that we'd hit it with the current configuration.)